### PR TITLE
Point to the conda-forge jaxlib wheels in the JAX readme.

### LINKED ISSUE #189: conda-based installation
Putting this here and tagging myself @ericmjl so that I can remember this exists.

To get `jax` into the hands of data scientists and machine learning researchers, `conda` installation would be very useful. I will take a stab at this on conda-forge, and record my progress here.

### PR DESCRIPTION
**Pull request by hawkinsp**
Fixes https://github.com/google/jax/issues/189

### CODE REVIEW

**Comment by hawkinsp** [comment]
@ngam @ericmjl @xhochy

**Review by jakevdp — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -512,6 +512,12 @@ jax.tools.colab_tpu.setup_tpu()
 Colab TPU runtimes use an older TPU architecture than Cloud TPU VMs, so installing `jax[tpu]` should be avoided on Colab.
 If for any reason you would like to update the jax & jaxlib libraries on a Colab TPU runtime, follow the CPU instructions above (i.e. install `jax[cpu]`).
 
+### Conda installation
+
+There is a community-supported Conda build of `jaxlib`. See the
+[jaxlib conda-forge](https://github.com/conda-forge/jaxlib-feedstock#installing-jaxlib)
+project for installation instructions.
+
```
[the remark this answers is not in the document]
Comment (hawkinsp) [reply README.md]: I incorporated a number of your suggestions. By the way, I found out that it is actually not sufficient to let `conda` install CUDA for you. JAX needs `ptxas`, which isn't distributed. So you must install at least the `ptxas` binary yourself. Everything else necessary is in the `conda` builds.

**Review by ngam — APPROVED** [review]

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (hawkinsp) [reply README.md]: The `ptxas` issue is fixable with a little bit of engineering work on our side (there's a library version of the same thing: `libnvptxcompiler_static.a`) or with the agreement of NVidia to allow redistribution of `ptxas`. I'll raise it with them.

[the remark this answers is not in the document]
Comment (gmarkall) [reply README.md]: Could this PTX Compiler Python binding be useful as an alternative to calling `ptxas`: https://github.com/rapidsai/ptxcompiler ?

[the remark this answers is not in the document]
Comment (hawkinsp) [reply README.md]: I can do you one better:

https://pypi.org/project/nvidia-cuda-nvcc-cu11/

**Pull request merged**